front/lib/models/tags.ts

```diff
@@ -22,6 +24,11 @@ TagModel.init(
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
+    reserved: {
```

Comment:
As always, free to ignore and go as is: 
-> To prepare for the future I think we might regret a boolean and will have more flexibility with a string enum field called status or whatever makes sense, I would do the refactor if I were you cause it's easier to do now 😄